src/base-test-class.js

```diff
@@ -122,7 +122,20 @@ BaseTest.prototype = {
               failed: o.failed
             };
           })
-        : null;
+        : {};
+
+        for(let key in this){
+          const val = this[key];
+          if(typeof val === 'function' && !['before', 'after', 'beforeEach', 'afterEach'].includes(key)){
+            let tc = testCases[key];
```

Comment:
Can either be `const` or rewritten as `if(!testCases[key])`

Reply:
updated